bakerydemo/base/fixtures/bakerydemo.json

```diff
@@ -2707,6 +2734,142 @@
       "alias_of": null
     }
   },
+  {
+    "model": "wagtailcore.page",
+    "pk": 80,
+    "fields": {
+      "path": "000100020005",
+      "depth": 3,
+      "numchild": 3,
+      "translation_key": "fa20c55f-7ea0-45a6-8d45-67695ea1ef81",
+      "locale": 1,
+      "latest_revision": null,
+      "live": true,
+      "has_unpublished_changes": false,
+      "first_published_at": "2019-02-10T16:25:47.179Z",
+      "last_published_at": null,
```

Comment:
I have backdated those pages and removed their revisions like we do for other pages.